src/Common/src/Interop/Linux/cgroups/Interop.cgroups.cs

```diff
@@ -79,6 +85,15 @@ private static bool TryReadMemoryValueFromFile(string path, out ulong result)
                         result = checked(ulongValue * multiplier);
                         return true;
                     }
+
+                    string value = Encoding.UTF8.GetString(bytes);
```

Comment:
We shouldn't need to get the string here; we can just use a span-based comparison on the bytes themselves.

Reply:
@stephentoub do you mean something like converting the string literal `max` to a byte array (using a decoding) and then doing `bytes.AsSpan() == max.AsSpan()`?

Reply:
Yes, e.g.
```C#
private static ReadOnlySpan<byte> MaxBytes => new byte[] { (byte)'m', (byte)'a', (byte)'x' };
...
if (MaxBytes.Equals(value, StringComparison.InvariantCultureIgnoreCase))
{
   ...
}
```
(While that MaxBytes looks like it's going to allocate every time, if you look at the IL you'll see the C# compiler doesn't actually do that.)